modules/navier_stokes/src/fvbcs/WCNSFVMomentumFluxBC.C

```diff
@@ -71,7 +71,12 @@ WCNSFVMomentumFluxBC::computeQpResidual()
     if (MooseUtils::absoluteFuzzyEqual(*_area_pp, 0))
       mooseError("Surface area is 0");
 
-  const Real contrib = std::abs(_face_info->normal()(_index));
+  // We assume that this boundary condition can only be specified on a boundary side.
```

Comment:
If we are going to assume this, then you should assert that the `_face_info->neighborPtr()` is `nullptr`

Reply:
Right, thanks for the review!